.evergreen/run-mongohouse-local.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+#
+# This script builds and launches a local mongohoused for testing.
```

Comment:
Is there a particular reason this isn't co-located in the `atlas_data_lake` directory? I think that would be consistent with what is done for some of the other projects included in the shared repository, and help keep the common path from becoming too cluttered.

Additionally, a basic README for the `atlas_data_lake` directory would be helpful. At the very least, it can cross-reference the tentative path that will be merged to the specifications repo in https://github.com/mongodb/specifications/pull/821 and briefly explain how this script would be integrated into a driver's Evergreen config.

See [`.evergreen/ocsp`](https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/ocsp) for some prior art. I realize that README doesn't do everything I asked for above, but it's still helpful in providing some context beyond just having a shell script and config file.